fe/fe-core/src/main/java/com/starrocks/sql/parser/SqlParser.java

```diff
@@ -37,7 +37,7 @@
                 statements.add((StatementBase) new AstBuilder().visitSingleStatement(sqlStatements.singleStatement(0)));
             } catch (ParsingException parsingException) {
                 StatementBase statement = parseWithOldParser(sql, session);
-                if (statement instanceof QueryStatement
```

Comment:
mistake before?

Reply:
Yes, this was an earlier bug. Because the old version of parser will not generate QueryStatement